client/src/pages/Lobby.jsx

```diff
@@ -43,7 +42,10 @@ export function Lobby() {
             : "Game loading"}
         </h3>
         <p className="mt-1 text-sm text-gray-500">
-          {`${currentPlayers} players have joined out of ${expectedPlayers} total expected players.`}
+          
+          {
```

Comment:
Hey @yiwei-ellen - I think you may have one more commit doesn't have the curly braces here - If you want to get that pushed, I'll approve the PR and you can merge it. =)